### PR TITLE
Added missing Belch tests

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

**Comment by AlexOn1ine** [comment]
I think assumes are missing

File: test/battle/move_effect/belch.c [hunk 1]
```diff
@@ -53,6 +53,58 @@ SINGLE_BATTLE_TEST("Belch cannot be used if the user has not eaten a berry")
     }
 }
 
-TO_DO_BATTLE_TEST("Belch can still be used after switching out");
-TO_DO_BATTLE_TEST("Belch can still be used after fainting");
-TO_DO_BATTLE_TEST("Belch can still be used after restoring the consumed berry");
+SINGLE_BATTLE_TEST("Belch can still be used after switching out")
+{
+    GIVEN {
+        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```
Comment (AsparagusEduardo) [inline test/battle/move_effect/belch.c:59]:
```suggestion
        ASSUME(gMovesInfo[MOVE_STUFF_CHEEKS].effect == EFFECT_STUFF_CHEEKS);
        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```

File: test/battle/move_effect/belch.c [hunk 2]
```diff
@@ -53,6 +53,58 @@ SINGLE_BATTLE_TEST("Belch cannot be used if the user has not eaten a berry")
...
+        SWITCH_OUT_MESSAGE("Greedent");
+        SWITCH_OUT_MESSAGE("Skwovet");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_BELCH, player);
+    }
+}
+
+SINGLE_BATTLE_TEST("Belch can still be used after fainting")
+{
+    GIVEN {
+        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```
Comment (AsparagusEduardo) [inline test/battle/move_effect/belch.c:78]:
```suggestion
        ASSUME(gMovesInfo[MOVE_STUFF_CHEEKS].effect == EFFECT_STUFF_CHEEKS);
        ASSUME(gMovesInfo[MOVE_FISSURE].effect == EFFECT_OHKO);
        ASSUME(gMovesInfo[MOVE_REVIVAL_BLESSING].effect == EFFECT_REVIVAL_BLESSING);
        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```

File: test/battle/move_effect/belch.c [hunk 3]
```diff
@@ -53,6 +53,58 @@ SINGLE_BATTLE_TEST("Belch cannot be used if the user has not eaten a berry")
...
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_REVIVAL_BLESSING, player);
+        SWITCH_OUT_MESSAGE("Skwovet");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_BELCH, player);
+    }
+}
+
+SINGLE_BATTLE_TEST("Belch can still be used after restoring the consumed berry")
+{
+    GIVEN {
+        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```
Comment (AsparagusEduardo) [inline test/battle/move_effect/belch.c:98]:
```suggestion
        ASSUME(gMovesInfo[MOVE_STUFF_CHEEKS].effect == EFFECT_STUFF_CHEEKS);
        ASSUME(gMovesInfo[MOVE_RECYCLE].effect == EFFECT_RECYCLE);
        PLAYER(SPECIES_GREEDENT) { Item(ITEM_ORAN_BERRY); }
```

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**